test/testground/manifest.toml

```diff
@@ -38,7 +38,7 @@ blobs_per_sequence = { type = "int", default = 1 }
 inbound_peer_count = { type = "int", default = 40 }
 outbound_peer_count = { type = "int", default = 10 }
 gov_max_square_size = { type = "int", default = 256 }
-max_block_bytes = { type = "int", deafult = 100000000 }
+max_block_bytes = { type = "int", default = 100000000 }
```

Comment:
wow I'm surprised the previous incorrect spelling didn't result in broken behavior

Reply:
I was surprised too, maybe max_block_bytes is not being test.. ?